### DISCUSSION TITLE
Two releases for different folders in the same repository.

### DISCUSSION THREAD

**Discussion by vishnu1002** [community/community#129711] [New to GitHub] [answered]
I'm experimenting with the new 'Releases' feature for my demo Node.js project. I've implemented two methods for running the project: one using npm locally and the other using Docker. I've uploaded these methods to GitHub in 'project-local' and 'project-docker' folders. How can I create separate releases for each of these folders? Thanks in advance!

**Comment by RachidZahrani** [comment, marked as answer]
To create separate releases for each folder in your Node.js project on GitHub:

1. **Tag the specific folders:**
   - Create a tag for the `project-local` folder:
     ```sh
     git tag -a v1.0-local -m "Release for local setup"
     git push origin v1.0-local
     ```
   - Create a tag for the `project-docker` folder:
     ```sh
     git tag -a v1.0-docker -m "Release for Docker setup"
     git push origin v1.0-docker
     ```

2. **Create Releases on GitHub:**
   - Go to the "Releases" section of your repository.
   - Click "Draft a new release."
   - Choose the tag (e.g., `v1.0-local` or `v1.0-docker`) and provide release notes.
   - Publish the release.

Repeat for each folder/tag to maintain separate releases for your `project-local` and `project-docker` setups.

**Reply by vishnu1002** [reply]
Thank you for your response! I’ve followed your advice, though I may have implemented it differently. Could you please review my work to see if it's correct? The project can be found here [https://github.com/vishnu1002/teamer-webapp](https://github.com/vishnu1002/teamer-webapp)
